recipe/meta.yaml

```
@@ -1,21 +1,25 @@
 {% set name = "soqt" %}
 {% set version = "1.6.0" %}
-{% set sha256 = "9f535af59f07c907022815679681bd345c9dec2f408c74833f6b1b24bca40e1f" %}
 
 package:
```

```suggestion
{% if build_variant == "qt6" %}
{% set name_suffix = "6" %}
{% else %}
{% set name_suffix = "" %}
{% endif %}

package:
```

then 
```
name: soqt{{ name_suffix }}
```

sigh, seems like we need to trick the template engine a little bit.

---

not sure if the above helps, but I would try that.